docker/jenkins2-centos/Dockerfile

```diff
@@ -9,7 +11,7 @@ RUN mv /usr/libexec/s2i/run /usr/libexec/s2i/run-base
 COPY ./contrib/jenkins /usr/local/bin
 
 # copy the s2i scripts. s2i expects them to be in /usr/libexec/s2i by default.
-ADD ./contrib/s2i /usr/libexec/s2i
+COPY ./contrib/s2i /usr/libexec/s2i
 
 # make the copied s2i scripts executable
 RUN chmod -R +x /usr/libexec/s2i/*
```

Comment:
Yeah, scripts that are made +x are copied in between. So, we cannot combine them. It is also not possible to move line 8 to line 17 since in that case scripts will be overwritten.